src/m365/spo/commands/list/list-list.ts

```diff
@@ -1,10 +1,10 @@
 import { Logger } from '../../../../cli/Logger';
 import GlobalOptions from '../../../../GlobalOptions';
-import request from '../../../../request';
+import { odata } from '../../../../utils/odata';
 import { validation } from '../../../../utils/validation';
 import SpoCommand from '../../../base/SpoCommand';
 import commands from '../../commands';
-import { ListInstanceCollection } from "./ListInstanceCollection";
```

Comment:
Same here for `ListInstanceCollection`. Do we still have need for this interface? I think this was the last location it was used.

Reply:
Correct!